checkbox-core-snap/prepare.sh

```diff
@@ -47,3 +47,5 @@ echo "Copying over checkbox-ng to $series"
 rsync -r --links ../checkbox-ng $series/
 echo "Copying over checkbox-support to $series"
 rsync -r --links ../checkbox-support $series
+echo "Dumping version in version file for $series"
+echo `git describe --tags --match "*[0-9]*" | grep -oP "(?<=v)\d+\.\d+"` > $series/version.txt
```

Comment:
a `grep -oP "(?<=v)\S+"` will support semantic versioning

Reply:
I have used that grep to strip any "dirty" tail from the calculated version. Removed now.